cmd/audit/audit.go

```diff
@@ -81,45 +82,60 @@ func read() error {
 	}
 
 	log.Debugln("starting netlink client")
-	client, err := libaudit.NewAuditClient(diagWriter)
-	if err != nil {
-		return err
-	}
 
-	status, err := client.GetStatus()
-	if err != nil {
-		return errors.Wrap(err, "failed to get audit status")
-	}
-	log.Infof("received audit status=%+v", status)
+	var err error
+	var client *libaudit.AuditClient
+	if *receiveOnly {
+		client, err = libaudit.NewMulticastAuditClient(diagWriter)
+		if err != nil {
+			return errors.Wrap(err, "failed to create receive-only audit client")
+		}
+	} else {
+		client, err = libaudit.NewAuditClient(diagWriter)
```

Comment:
As far as I understand, all the checks below are not needed in the case of the MultiCase client?